docs/index.md

```diff
@@ -757,7 +757,70 @@ If you use a virtualenv for your current project, add a path to it in your [proj
 }
 ```
 
-See: [github:palantir/python-language-server](https://github.com/palantir/python-language-server)
+A basic configuration below can be used for bootstraping your own:
+
+```js
+  //...
+"pyls": {
+  "enabled": true,
+  "command": ["pyls"],
+  "languageId": "python",
+  "scopes": ["source.python"],
+  "syntaxes": [
+    "Packages/Python/Python.sublime-syntax",
+    "Packages/MagicPython/grammars/MagicPython.tmLanguage",
+    "Packages/Djaneiro/Syntaxes/Python Django.tmLanguage",
+  ],
+  "settings": {
+    "pyls": {
+      "env":
+      {
+        // Making Sublime's own libs available to the linters.
+        // "PYTHONPATH": "/Applications/Sublime Text.app/Contents/MacOS/Lib/python33",
+      },
+      // Configuration is computed first from user configuration (in home directory),
+      // overridden by configuration passed in by the language client,
+      // and then overridden by configuration discovered in the workspace.
+      "configurationSources": [
+        "pycodestyle",  // discovered in ~/.config/pycodestyle, setup.cfg, tox.ini and pycodestyle.cfg
+        // "flake8",  // discovered in ~/.config/flake8, setup.cfg, tox.ini and flake8.cfg
+      ],
+      "plugins": {
+        "jedi": {
+          "extra_paths": [
+            // The directory where the pip installation package is located
+          ],
+        },
+        "jedi_completion": {
+          "fuzzy": false,  // Enable fuzzy when requesting autocomplete
```

Comment:
Set this to `true` :) It probably works well with ST's own fuzzy string matcher for completions.

Reply:
Will do but it seems fuzzy to me even without it:

![Screenshot 2020-05-11 at 10 56 56](https://user-images.githubusercontent.com/153197/81543327-33a38580-9376-11ea-98ad-5dac3c7803dc.png)

I've triggered this completion on typing `m` in `fnm`.